dropwizard-hibernate/src/test/java/io/dropwizard/hibernate/JerseyIntegrationTest.java

```diff
@@ -117,11 +117,12 @@ protected Application configure() {
                                             ImmutableList.of(Person.class));
 
         try (Session session = sessionFactory.openSession()) {
-            session.createSQLQuery("DROP TABLE people IF EXISTS").executeUpdate();
-            session.createSQLQuery(
+            session.beginTransaction();
```

Comment:
I think if we open a transaction manually, it makes sense to commit manually too,